packages/validator/tsconfig.build.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../tsconfig.build.json",
   "compilerOptions": {
     "baseUrl": "./",
-    "outDir": "./dist"
+    "outDir": "./dist/src"
```

Comment:
I'm a bit worried that if you do this the imports will be different for local imports and imports from the package version (because the packaged version will have all the code under `src` and not directly, basically one level deeper).

But feel free to leave it as it is for now, I'll be going through all the packages anyway when I'll be sorting packaging.

Reply:
Im not sure I follow this, the source of validator reads all of the templates relatively, which means if a file is located in `src/` it would look in `../templates/`. I thought package would consist only of the contents of `dist/`, and thats why the build was moved to `dist/src/` and templates are copied to `dist/templates/` so that package can access templates the exact same way.

Reply:
I wasn't talking about imports within the validator but about some other package importing validator. But in the meantime I was looking more into it as I'm doing the node dockerization and it looks like it shouldn't be a problem after all.

Reply:
Yeah I wasnt talking about imports as well, I found that reading the templates might be a problem when I was working on the integration with deployer. I updated validator so that it always resolves paths relative to the current file, so even if it's imported and ran from completely different directory it will be reading the correct directory. I'm not 100% sure if this will work as package, but it is javascript after all so hopefully it wont need any more adjustments, but we can fix this if it becomes a problem in packaging.